egs/librispeech/ASR/prepare.sh

```diff
@@ -19,8 +18,53 @@ fi
 if [ $stage -le 0 ] && [ $stop_stage -ge 0 ]; then
   echo "stage 0: Download data"
 
-  # If you have pre-downloaded it in /path/to/LibriSpeech
-  # Just run: ln -sfv /path/to/LibriSpeech data/
+  # If you have pre-downloaded it to /path/to/LibriSpeech,
+  # you can create a symlink to avoid downloading it again:
+  #
+  #   ln -sfv /path/to/LibriSpeech data/
+  #
+
   mkdir -p data/LibriSpeech
-  # TODO
+
+  if [ ! -f data/LibriSpeech/train-other-500/.completed ]; then
+    # It's compatible with kaldi's egs/librispeech/s5/local/download_and_untar.sh
+    ./local/download_data.py
+  fi
+
+  # If you have pre-downloaded it to /path/to/musan,
+  # you can create a symlink to avoid downloading it again:
+  #
+  #   ln -s /path/to/musan data/
+  #
+  if [ ! -e data/musan ]; then
+    wget https://www.openslr.org/resources/17/musan.tar.gz
```

Comment:
There is `download_musan()` here: https://github.com/lhotse-speech/lhotse/blob/master/lhotse/recipes/musan.py#L27

and `$ lhotse download musan` here: https://github.com/lhotse-speech/lhotse/blob/master/lhotse/bin/modes/recipes/musan.py#L25